tests/e2e/utils/OpenshiftHelper.ts

```diff
@@ -0,0 +1,117 @@
+import { echo, exec, ShellString } from 'shelljs';
+import { TestConstants } from '../constants/TestConstants';
+import { Logger } from './Logger';
+
+export class OpenshiftHelper {
```

Comment:
This is currently `OpenshiftClientCommandRunner` class, because is running oc client command.

Reply:
changed to OpenshiftClientExecutor